### ISSUE TITLE
Migrations generator assumes migrations are in db/migrate

### ISSUE THREAD

**Issue by zachmargolis** [bensheldon/good_job#352]
## tl;dr

Would like `bin/rails g good_job:update` to allow us to specify a separate DB or separate migrations directory, like the `--database` option in `rails generate migration`

## Background

In our app, we have set up the good_jobs table in a separate database from our main application.

Per the [ActiveRecord multiple databases guide](https://guides.rubyonrails.org/active_record_multiple_databases.html#generators-and-migrations), we use the `migrations_paths` config to put those migrations are in `db/worker_jobs_migrate` instead of the normal `db/migrate`.

When generating migrations, we use the `--database` CLI arg to specify `worker_jobs` so Rails knows to put things in the right spot, ex:

```
bin/rails generate migration CreateDogs name:string --database worker_jobs
```

However, GoodJob assumes migrations are in `db/migrate`:
https://github.com/bensheldon/good_job/blob/236aea27f3217e1132302804984ed269a4f5224f/lib/generators/good_job/update_generator.rb#L26

## Problems

Since our migrations are in a different place, GoodJob is unable to detect migrations that have already happened to give us diffs

**Comment by bensheldon** [comment]
Released in `v1.99.1` and `v2.0.2`. Thank you!

**Comment by zachmargolis** [comment]
Such a fast turnaround! Thank you! 🙌 😍
